README.md

```diff
@@ -14,9 +15,9 @@ Building The Provider
 
 1. Clone the repository
 1. Enter the repository directory
-1. Build the provider using the Go `install` command: 
+1. Build the provider using the Go `make install_macos` command: 
```

Comment:
Should we make this a generic (without os-specific) build information like `make` or `make build` and in the `developing the provider` section, we have the MacOS-specific instructions?

Reply:
This is now handled through the make file and a build target.